types/aws-lambda/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for AWS Lambda 8.10
+// Type definitions for AWS Lambda 8.11
```

Comment:
Revert this, the idea is these are definitions for the node 8.10 lambda runtime. Yeah, it's confusing when external services add features, but at least there's some version to tie this to.

Reply:
Will do.